templates/pesto/data/recipes.json

```diff
@@ -0,0 +1,940 @@
+{
+  "recipes" :[
+    {
+      "title": "Cake",
+      "author": "Johnny Appleseed",
+      "description": "Delicious cake will be offered at the end of the test. It is not a lie.",
+      "imageUrl": "/components/app-layout/templates/pesto/img/img1.jpg",
```

Comment:
The path should be relative to `app-layout`, e.g. `img/img1.jpg`.  Same goes to the other paths in this file and the `background-image` in `recipe-list.html` (https://github.com/PolymerLabs/app-layout/pull/22/files#diff-41a0fae865f33fe1cf658e543e7afd33R33)